src/js/components/stock-movement-wizard/EditPage.jsx

```diff
@@ -130,7 +130,7 @@ const FIELDS = {
           productCode: fieldValue.productCode,
           btnOpenText: `react.stockMovement.${fieldValue.substitutionStatus}.label`,
           btnOpenDefaultText: `${fieldValue.substitutionStatus}`,
-          btnOpenDisabled: fieldValue.substitutionStatus === 'NO' || fieldValue.statusCode === 'SUBSTITUTED' || showOnly,
+          btnOpenDisabled: fieldValue.statusCode === 'SUBSTITUTED' || showOnly,
           btnOpenClassName: BTN_CLASS_MAPPER[fieldValue.substitutionStatus || 'HIDDEN'],
```

Comment:
If the substitutionStatus will be 'NO', then the css class "disabled" will be applied.

Reply:
It should stay grey when there is no substitution but should be clickable.

Reply:
Grey comes from btn-outline-secondary not from disabled (it adds small opacity). Disabled class also affects the cursor. 
>.btn:not(:disabled):not(.disabled) {
    cursor: pointer;
}

Please remove disabled class.

Reply:
Done